ephios/core/signup/structure/abstract.py

```diff
@@ -49,12 +52,22 @@ def disposition_participation_form_class(self):
         """
         raise NotImplementedError()
 
-    @property
-    def signup_form_class(self):
-        """
-        This form will be used for participations in signup.
-        """
-        raise NotImplementedError()
+    def get_signup_form_fields(
+        self,
+        participant: AbstractParticipant,
+        participation: AbstractParticipation,
+        signup_choice,
+    ):
+        pass
+
+    def save_signup(
+        self,
+        shift: Shift,
+        participant: AbstractParticipant,
+        participation: AbstractParticipation,
+        cleaned_data: dict[str, Any],
```

Comment:
cleaned data does contain signup_choice, right? I think it might be nice to explicitly have it as kwargs so this methods signature is an extension of get_signup_form_fields